pyproject.toml

```diff
@@ -3,17 +3,23 @@ requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "hello_world"
-description = "Prints messages to stdout. Simple boilerplate for libraries."
+name = "code_ocean_api"
+description = "Code ocean API manager in Python"
 license = {text = "MIT"}
 requires-python = ">=3.7"
+authors = [
+    {name = "David Feng", email="david.feng@alleninstitute.org"},
+    {name = "Jon Young", email="jon.young@alleninstitute.org"},
+    {name = "Dana Rocha", email="dana.rocha@alleninstitute.org"},
+    {name = "Camilo Laiton", email="camilo.laiton@alleninstitute.org"}
```

Comment:
@dyf Did we come to a consensus on whether to use individual authors or the institution?

Reply:
(I lean towards using the institution)

Reply:
Let's be sure to include the institute (`Allen Institute for Neural Dynamics`), with no email.

Others can opt-in to including their individual names/emails.